subsys/emds/emds_flash.c

```diff
@@ -530,19 +559,21 @@ int emds_flash_prepare(struct emds_fs *fs, int byte_size)
 		return -ENOMEM;
 	}
 
-	int rc = old_entries_invalidate(fs);
+	err = old_entries_invalidate(fs);
 
-	if (rc) {
-		return rc;
+	if (err < 0) {
+		return err;
 	}
 
+	LOG_ERR("Byte size: %d, Free space: %d", byte_size, emds_flash_free_space_get(fs));
 	if (fs->force_erase || (byte_size > emds_flash_free_space_get(fs))) {
 		emds_flash_clear(fs);
 		fs->force_erase = false;
+		err = 1;
```

Comment:
Is this an error, misliding variable name "err". Would be better with rc, as return code.

Reply:
Agree 👍